src/Build/Logging/BinaryLogger/ProjectImportsCollector.cs

```diff
@@ -140,11 +142,13 @@ bool TryAddFile()
             {
                 try
                 {
-                    addFileWorker(filePath);
+                    addFileWorker(filePath); 
                     return true;
                 }
-                catch
-                { }
+                catch (Exception e)
+                {
+                    InvokeFileIOErrorEvent(filePath, e.Message);
```

Comment:
It'd be good to collect chain of all nested exception messages.
@JaynieBai I recall you created some helper for that - would you be able to point to location?

Reply:
Is that necessary? e.ToString() basically does this